### PR TITLE
Expand the speculation rules explainer.

### PR DESCRIPTION
**Pull request by jeremyroman**
/cc @buettner @kenjibaheux

### CODE REVIEW

File: triggers.md [hunk 1]
```diff
@@ -1,18 +1,68 @@
...
+
+Accordingly, we need a trigger that is flexible enough to accommodate these and other, unanticipated, future needs. It should also **fail safe**, meaning that if new complications are added in the future, it should default to *not* making requests which may not be intended, rather thaan failing open (e.g., erroneously issuing fetches due to having ignored unsupported syntax).
+
+## Non-goals
+
+While we intend to better specify the behavior of "prefetch", "prerender" and similar speculated actions, these specifications are largely separable from the trigger API itself. In fact, this specified behavior should be shared as much as possible between existing resource hints, those proposed in this document, and browser features for prefetching and prerendering.
+
+This proposal doesn't aim to address concerns about any particular provider, service or software a browser may use to provide IP anonymization. Feedback on [private prefetch proxies](https://github.com/buettner/private-prefetch-proxy/issues) is welcome, but tracked separately.
+
+## Alternatives considered
```
Comment (domenic) [inline triggers.md:53]: It's a little strange to state the alternative before the proposed solution. I'd put this at the bottom? Or, move the "Speculation rules" section above, and the move all of its subsections below into "Details" or something.

**Review by domenic — APPROVED** [review]
Text LGTM, just a structural concern.

**Review by domfarolino — APPROVED** [review]

File: triggers.md [hunk 2]
```diff
@@ -1,18 +1,68 @@
...
+
+The obvious alternative is to extent the `<link>` element.
```
Comment (buettner) [inline triggers.md:55]: extent -> extend

**Comment by buettner** [comment]
LGTM

File: triggers.md (hunk 2 above)
Comment (jeremyroman) [reply triggers.md:55]:
```suggestion
The obvious alternative is to extend the `<link>` element.
```

File: triggers.md (hunk 1 above)
Comment (jeremyroman) [reply triggers.md:53]: Moved to bottom and re-ran doctoc.

**Review by domenic — APPROVED** [review]

**Pull request merged**